docs/src/appendix/sncast/common.md

```diff
@@ -3,7 +3,8 @@
 ## `--profile, -p <PROFILE_NAME>`
 Optional.
 
-Profile name in `Scarb.toml` config file.
+Used for both `snfoundry.toml` and `Scarb.toml` if specified (defaulting to `dev` in `Scarb.toml` if absent).
+Defaults to `default`.
```

Comment:
Not sure if i understand. First sentence is about the being specified case, second about the default, but you put parentheses with additional information there.

Reply:
this is not trivial to explain in one sentence, so maybe I did too big somersault here.

What I wanted to include in this iformation is:

```
if profile is passed, use it in sncast and scarb, unless there is no such profile in scarb, then `dev` will be used.
If profile is not passed, use `default` in cast and `dev` in scarb
```

I feel somethings like this:

```
Used for both `snfoundry.toml` and `Scarb.toml` if specified.
Defaults to `default` (`snfoundry.toml`) and `dev` (`Scarb.toml`).
```

Does not fully explains it. any ideas?

Reply:
Yes

Reply:
That's cool, do you mind sharing? :D

Reply:
Sorry didn't sleep well today :D , i read it as 

> Does it fully explain the idea?

I think what you suggested is sufficient.

Reply:
But you can also make it as you specified above. I don't think there is any hurt in verbosity.

Reply:
done https://github.com/foundry-rs/starknet-foundry/pull/1647/commits/173cdcf3660f62ed54621c4f138a0caf0c0e3f70